### PR TITLE
Added new markdown viewer for IC about page

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Added new markdown viewer since [simpleMDE](https://github.com/sparksuite/simplemde-markdown-editor?tab=readme-ov-file#how-it-works) uses [marked](https://github.com/markedjs/marked), so the editor and viewer now has the consistency.
image

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Megha-Dev-19 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Do you want to apply for monthly streak? Get 8+ score for a single PR this month and receive boost for race-of-sloths!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Megha-Dev-19)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Megha-Dev-19)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: instances/devhub.near/widget/devhub/components/molecule/SimpleMDEViewer.jsx
Comment (petersalomonsen) [inline instances/devhub.near/widget/devhub/components/molecule/SimpleMDEViewer.jsx:52]: I believe also this script could send a message about its content height, so that the parent widget can adjust the height attribute according to that

File: instances/devhub.near/widget/devhub/components/molecule/SimpleMDEViewer.jsx
Comment (petersalomonsen) [inline instances/devhub.near/widget/devhub/components/molecule/SimpleMDEViewer.jsx:63]: So the height attribute here should be set with the value from the message sent from the iframe content

File: instances/infrastructure-committee.near/widget/components/admin/AboutConfigurator.jsx
Comment (petersalomonsen) [inline instances/infrastructure-committee.near/widget/components/admin/AboutConfigurator.jsx:87]: This might be redundant if the iframe height is set

File: playwright-tests/tests/infrastructure/infrastructure.near.spec.js
Comment (petersalomonsen) [inline playwright-tests/tests/infrastructure/infrastructure.near.spec.js:13]: Maybe also add some expectation about not seeing the scrollbar for longer content

**Comment by petersalomonsen** [comment]
@race-of-sloths score 3

**Review by petersalomonsen — APPROVED** [review]
Consider my comments about setting the height, or we'll address it later.

**Pull request merged**
